crates/secrets-nats-kv/src/lib.rs

```diff
@@ -5,4 +5,4 @@ pub mod types;
 pub use types::*;
 
 pub mod client;
-pub use client::*;
+// pub use client::*;
```

Comment:
Is this intentionally left commented out?

Reply:
Nope